### PR TITLE
V1.0.0 changes

### PR DESCRIPTION
**Pull request by jessiedeot**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,6 +34,24 @@ For all changes follow the procedure documented at the bottom of the
...
+
+The CLI tests only test that the CLI completes and returns exit status 0 (success).  The CLI tests do not test the
+functionality of the CLI.  This should be sufficient since the functionality is already covered in the Python Test cases.
+
+### Code style
+
+The project style guide is [PEP8](https://www.python.org/dev/peps/pep-0008/).  Flake8 is a useful tool to test whether
+the PEP8 standard has been met (although this testing has not been automated yet).
+
+Another useful resource is the [Google Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md) which
```
Comment (cwhanse) [inline CONTRIBUTING.md]: I think we agreed to use the Google style for docstrings. 

`Docstrings follow the [Google Style Guide]` which also has additional guidance for coding above and beyond PEP8.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,6 +34,24 @@ For all changes follow the procedure documented at the bottom of the
...
+has additional details about writing coding above and beyond what is available in pep8.
+
 ## How to report a bug
 
 Please use [GitHub Issues](https://github.com/SunSpecOrangeButton/pyoblib/issues) for all bug reports.
```
Comment (cwhanse) [inline CONTRIBUTING.md]: `bug reports and requests for specific code improvements.  User questions, suggestions for improvement and feature requests can be raised at ` insert link to our user group

File: CONTRIBUTING.md (hunk 1 above)
Comment (jessiedeot) [reply CONTRIBUTING.md]: This change has been incorporated.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jessiedeot) [reply CONTRIBUTING.md]: Link to Slack is available in the`How to suggest a feature or enhancement` section

**Pull request merged**